src/med_bench/utils/classifiers.py

```diff
@@ -0,0 +1,508 @@
+from itertools import combinations
```

Comment:
To be removed

Reply:
This has been addressed in commit 14374c26028c5a15b5ce4dd3e8cd907a83d83a37